src/main/src/test/java/org/geoserver/logging/LoggingStartupContextListenerTest.java

```diff
@@ -174,4 +175,36 @@ public void testFilenameAndFilePattern() throws Exception {
                 GeoServerXMLConfiguration.applyPathTemplate(
                         "${baseDir}/logs/logging.log", "logs/geoserver.log"));
     }
+
+    @Test
+    public void testNodeAttributes() {
+        Node node = new Node();
+        node.getAttributes().put("fileName", "logs/geoserver.log");
+
+        assertEquals("fileName", GeoServerXMLConfiguration.attribute(node, "fileName"));
```

Comment:
Something similar to this should work too, and avoid the need for a dedicated method?

```suggestion
        assertNotNull(GeoServerXMLConfiguration.attributeGet(node, "fileName"));
```

Reply:
Got it, double check that attributeGet is case in-sensitive and yes combine the methods.